agent/bench-scripts/test-bin/which

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
It would be worthwhile having a header comment explaining what the purpose of this script is.  (I presume it is intended to direct an invocation of `which` to the `/usr/bin/which` command instead of letting it be intercepted by a shell function or alias, and it also ignores any command line switches.)